lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -23,20 +23,25 @@ class DescribeMethod < Base
         MSG = 'The second argument to describe should be the method ' \
               "being tested. '#instance' or '.class'."
 
-        # @!method second_argument(node)
-        def_node_matcher :second_argument, <<~PATTERN
+        # @!method describe_method(node)
+        def_node_matcher :describe_method, <<~PATTERN
```

Comment:
`describe_method` seems to be more like "describe with a second argument that does not resemble a method name", quite the opposite of its name.

Reply:
Thanks for catching this one too. I refactored it.